tests/utils/initial-state.json

```diff
@@ -30,5 +30,7 @@
     "demandFactor": 1,
     "revenueThisPeriod": 0,
     "consecutivePeriodsWithMinDemandFactor": 0
-  }
+  },
+  "prescribedObservers": {},
+  "lastTickedHeight": 0
```

Comment:
this not being in here was a bug - interactions weren't ticking state before executing, hence the changes to various tests